src/main/java/org/cascadebot/cascadebot/events/ModlogEventListener.java

```diff
@@ -409,12 +409,8 @@ private SerializableMessage getMessageFromString(long id, String messageString)
             }
             try {
                 message = CryptUtils.decryptString(Config.INS.getEncryptKey(), iv, messageBytes);
-            } catch (NoSuchPaddingException | NoSuchAlgorithmException | InvalidAlgorithmParameterException | InvalidKeyException | IllegalBlockSizeException | ShortBufferException e) {
-                e.printStackTrace();
-                // TODO log these
-                return null;
-            } catch (BadPaddingException e) {
-                e.printStackTrace();
+            } catch (NoSuchPaddingException | NoSuchAlgorithmException | InvalidAlgorithmParameterException | InvalidKeyException | IllegalBlockSizeException | ShortBufferException | BadPaddingException e) {
+                CascadeBot.LOGGER.error("Unable to decrypt message!", e);
```

Comment:
I made the BadPaddingException a separate block as thats happens when our redis has been access and data messed with which we should probably know about. I should have documented that that was the case in hindsight.